frontend/shared/src/error-handler/error-handler.ts

```diff
@@ -6,16 +6,21 @@ import { isError } from 'shared/utils/type-guards';
 type Params = {
   error: Error | unknown;
   t: TFunction;
+  pathname: string;
   goToPage: GoToPageFunction;
   onAuthError?: () => void;
   onServerError?: () => void;
   onCommonError?: () => void;
 };
 
 const handleError = (args: Params): void => {
-  const { error, t, goToPage } = args;
+  const { error, t, pathname, goToPage } = args;
   const {
-    onAuthError = () => goToPage('/login?sessionExpired=true'),
+    onAuthError = () => {
+      if (pathname !== '/login') {
```

Comment:
Is it so that the `pathname` here does not include query parameters so even if we're already on page `/login?sessionExpired=true` the pathname will be `/login`?

Reply:
It doesn't

Reply:
Ok, good